src/main/java/ro/societateahermes/backendservice/service/serviceImplementation/ActivityServiceImplementation.java

```diff
@@ -0,0 +1,68 @@
+package ro.societateahermes.backendservice.service.serviceImplementation;
+
+import org.springframework.stereotype.Service;
+import ro.societateahermes.backendservice.entities.Activity;
+import ro.societateahermes.backendservice.entities.DTO.ActivityDTO;
+import ro.societateahermes.backendservice.entities.User;
+import ro.societateahermes.backendservice.repository.ActivityRepositoryInterface;
+import ro.societateahermes.backendservice.service.ActivityServiceInterface;
+
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+
+@Service
+public class ActivityServiceImplementation implements ActivityServiceInterface {
+    private final ActivityRepositoryInterface activityRepository;
+
+    public ActivityServiceImplementation(ActivityRepositoryInterface activityRepository) {
+        this.activityRepository = activityRepository;
+    }
+
+    @Override
+    public void save(ActivityDTO activity) {
+
+        Activity updatedActivity = new Activity();
+        updatedActivity.setActivityLink(activity.getActivityLink());
```

Comment:
Convertirea unui dto in entitate ar trebui sa fie facuta printr-o clasa separata. un UserMapper, nu in service. Service-ul nu trebuie sa stie sa converteasca din dto in entitate si invers